### PR TITLE
Move reset user from admin to Dashboard API

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/tyk-dashboard-api/users.md [hunk 1]
```diff
@@ -272,6 +272,43 @@ authorization:7a7b140f-2480-4d5a-4e78-24049e3ba7f8
...
+| Resource URL | `/api/users/{USER_ID}/actions/key/reset`   |
+| Method       | PUT                                        |
+| Type         | None                                       |
+| Body         | {"userId":"{USER_ID}"}                     |
+| Param        | None                                       |
+
+#### Sample Request
+
+```{.copyWrapper}
+PUT /api/users/54c25e845d932847067402e2/actions/key/reset HTTP/1.1
```
Comment (sedkis) [inline tyk-docs/content/tyk-dashboard-api/users.md:294]: Please replace this ID with {USER_ID} to match the body below

File: tyk-docs/content/tyk-dashboard-api/users.md [hunk 2]
```diff
@@ -272,6 +272,43 @@ authorization:7a7b140f-2480-4d5a-4e78-24049e3ba7f8
...
+Host: localhost:3000
+admin-auth: 12345
```
Comment (sedkis) [inline tyk-docs/content/tyk-dashboard-api/users.md]: This needs to be changed from "admin-auth" to match the other APIs in this section.

File: tyk-docs/content/tyk-dashboard-api/users.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/tyk-dashboard-api/users.md:294]: I left the ID as in the other request samples on this page as others wanted a "proper" example.

File: tyk-docs/content/tyk-dashboard-api/users.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/tyk-dashboard-api/users.md]: Done

**Review by sedkis — APPROVED** [review]

**Pull request merged**